systems/rolls.md

```diff
@@ -12,10 +14,86 @@ Disadvantage is a `-1d4` that is added to your roll. See [advantage](#Advantage)
+
+On a critical, the character may select an applicable critical hit effect. All characters have [pierce defence](#Pierce-defence) as a critical effect option by default. Multiple critical effects may be selected for each pair of duplicates. Three of a kind count as two criticals, and so on.
+
+If you have a [disadvantage](#Disadvantage) (That is, more disadvantages than advantages) you are **not elegable for criticals**. You may instead be elegible for [critical failures](#Critical-failure).
+
+## Critical failure
+
+Critical failures occurr when you roll a critical, but have [disadvantage](#Disadvantage). This event is usually ignored, but may be mentioned in specific rules or features.
+
+## Pierce defence
+
+This is a [critical](#Criticals) effect that all characters have access to by default. This allows you to ignore the foes [defence](stats.md#Defence) when calculating damage.
+
+### Example 1:
+
+A character performs an attack with a greatsword (`d10`) and `+4` strength. Because the foe is both [flanked](statuses.md#Flanked) and [marked](statuses.md#Marked), he also has `2x` [advantage](#Advantage).
+
+This means he rolls `2d10 + 4 + 2d4` for his attack score. His die are `8,3,3,2`. Because there are two `3`'s, he has rolled a critical. The character chooses to use this critical to [pierce defence](#Pierce-defence), ignoring the foes [defence](stats.md#Defence).
+
+The target then takes the entire `20` damage.
+
+### Example 2:
+
+A character performs an attack with a hammer `d8`, and a strength of `+2`.
+
+This means he rolls `2d8 + 2` for his attack score. His die are `2,2`. Because there are two `6`'s, he has rolled a critical.
+
+Because his total damage score of `6`, and his foes [defence](stats.md#Defence) is `7`, this is not a hit. Becase his [stunning blow](feats.md#Stunning-blow) feat requires a hit, he may **not** use it. Instead, he will [pierce defence](#Pierce-defence), and deal `6` damage.
+
+### Example 3:
+
+A character performs an attack with a dagger `d4` and a agility of `+3`. Because he is [hidden](statuses.md#Hidden) and the foe is [flanked](statuses.md#Flanked) he as `2x` [advantage](#Advantage).
+
+This means he rolls `4d4 + 3` for his attack score. His die are `1,1,1,3`. This is a double critical, and he may have two critical effects. He chooses to both [pierce defence](#Pierce-defence) and [twist the knife](feats.md#Twist-the-knife) for a bonus `1d8`.
+
+Rolling `6` for twist the knife, `15` damage is dealt in total.
+
+### Example 4
+
+A character performs an attack with a shortsword `d6` and a strength of `+1`. Because his foe is [in cover](statuses.md#In-cover), he is rolling with [disadvantage](#Disadvantage).
+
+He rolls `6,6,-2`, for a total of `11` damage. Due to his disadvantage, this is not a [critical failure](#Critical-failure). Because there are no features active that are triggered from a critical falure, this is ignored.
```

Comment:
Total of 10 damage

Reply:
This is 11 damage once the strength of +1 is added. Should I make this more clear?

Reply:
Ahh, I must be tired. I guess it's easy to miss because there is no formula. It's probably fine